pkg/client/unversioned/request_test.go

```diff
@@ -959,9 +962,6 @@ func TestDoRequestNewWayFile(t *testing.T) {
 	requestURL := testapi.Default.ResourcePathWithPrefix("foo/bar/baz", "", "", "")
 	requestURL += "?timeout=1s"
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &tmpStr)
-	if fakeHandler.RequestReceived.Header["Authorization"] == nil {
```

Comment:
There are other occurrences in this file (e.g., L864). Why not removing them altogether?